packages/docusaurus-preset-classic/src/index.js

```diff
@@ -18,8 +18,8 @@ module.exports = function preset(context, opts = {}) {
       algolia && '@docusaurus/theme-search-algolia',
     ],
     plugins: [
-      ['@docusaurus/plugin-content-docs', opts.docs],
       ['@docusaurus/plugin-content-blog', opts.blog],
+      ['@docusaurus/plugin-content-docs', opts.docs],
```

Comment:
The blogs should load first to provide the blogs routes priority in case of a conflict.

How can there be a conflict?
- When the docs **path** is set to "." (the root of the project).
- A **blog** directory exists in the root of the project. (This will setup a router path for "/blog" when the docs plugin loads).
- The blog plugin is enabled and it's **routeBasePath** is set to "blog".